README.md

```diff
@@ -4,7 +4,16 @@
 
 Binding generator for [FFI](https://dart.dev/guides/libraries/c-interop) bindings.
 
-> Note: ffigen only supports parsing `C` headers.
+> Note: ffigen only supports parsing `C` headers, not `C++` headers.
```

Comment:
I believe we should the C++ note, otherwise developers are going to ask questions again.

Reply:
Oh, you're adding it. I can't read diffs. 😄